Mathlib/Algebra/Order/AbsoluteValue/Equivalence.lean

```diff
@@ -0,0 +1,53 @@
+/-
+Copyright (c) 2025 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import Mathlib.Analysis.SpecialFunctions.Pow.Real
+
+/-!
+# Equivalence of real-valued absolute values
+
+Two absolute values `v₁, v₂ : AbsoluteValue R ℝ` are *equivalent* if there exists a
+positive real number `c` such that `v₁ x ^ c = v₂ x` for all `x : R`.
+-/
+
+namespace AbsoluteValue
+
+variable {R : Type*} [Semiring R]
+
+/-- Two absolute values `f, g` on `R` with values in `ℝ` are *equivalent* if there exists
+a positive real constant `c` such that for all `x : R`, `(f x)^c = g x`. -/
+def equiv (f g : AbsoluteValue R ℝ) : Prop :=
+  ∃ c : ℝ, 0 < c ∧ (f · ^ c) = g
+
+/-- Equivalence of absolute values is reflexive. -/
+lemma equiv_refl (f : AbsoluteValue R ℝ) : equiv f f :=
+  ⟨1, Real.zero_lt_one, funext fun x ↦ Real.rpow_one (f x)⟩
+
+/-- Equivalence of absolute values is symmetric. -/
+lemma equiv_symm {f g : AbsoluteValue R ℝ} (hfg : equiv f g) : equiv g f := by
+  rcases hfg with ⟨c, hcpos, h⟩
+  refine ⟨1 / c, one_div_pos.mpr hcpos, ?_⟩
+  simp [← h, Real.rpow_rpow_inv (apply_nonneg f _) (ne_of_lt hcpos).symm]
+
+/-- Equivalence of absolute values is transitive. -/
+lemma equiv_trans {f g k : AbsoluteValue R ℝ} (hfg : equiv f g) (hgk : equiv g k) :
+    equiv f k := by
+  rcases hfg with ⟨c, hcPos, hfg⟩
+  rcases hgk with ⟨d, hdPos, hgk⟩
+  refine ⟨c * d, mul_pos hcPos hdPos, ?_⟩
+  simp [← hgk, ← hfg, Real.rpow_mul (apply_nonneg f _)]
+
+/-- An absolute value is equivalent to the trivial iff it is trivial itself. -/
+lemma eq_trivial_of_equiv_trivial [DecidablePred fun x : R ↦ x = 0] [NoZeroDivisors R]
```

Comment:
Should this be a simp lemma?

Reply:
Probably. At least the rhs looks simpler...